apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/domain-step/index.tsx

```diff
@@ -41,14 +45,26 @@ const DomainStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep, o
 		} );
 	};
 
+	const fallbackSubtitleText = __(
+		'Free for the first year with any paid plan.',
+		'full-site-editing'
+	);
+	const newSubtitleText = __(
+		'Free for the first year with any annual plan.',
+		'full-site-editing'
+	);
+	const subtitleText =
+		locale === 'en' ||
+		hasTranslation?.( 'Free for the first year with any annual plan.', 'full-site-editing' )
```

Comment:
Strange that `hasTranslation` doesn't automatically return `true` for English (I checked).

Reply:
I believe that's by design, since we consider the EN strings as originals and not translations 🤷🏼‍♂️

Reply:
That makes some sense considering the function name, but `hasTranslations` doesn't accept a locale. So you, as an English user, can't use it to check if the phrase is available in Spanish for example. The only use case that I understand is whether a given phrase good to show to the user in the current locale, and the phrase is good to show when it's English.

Reply:
I agree the implementation is not ideal, but I still think it makes sense for it to be bound to the current locale, since both the react bindings and the i18n library itself are designed to work with a single locale data a time.
As for the use case that you described, that would be a matter of creating another i18n instance with the target locale data and use its methods where needed.